src/layman/map/rest_maps.py

```diff
@@ -102,7 +102,7 @@ def post(username):
         input_file.save_map_files(
             username, mapname, [file])
 
-        actor_name = g.user and g.user["username"]
+        actor_name = g.user and g.user.get("username")
```

Comment:
Skoro si říkám, jestli to není na jednu funkci, která zaobalí "g.user and g.user.get("username")".

Reply:
Asi jo, přidal jsem